cla_public/templates/accessibility-statement.html

```diff
@@ -7,6 +7,7 @@ <h1 class="govuk-heading-xl">{{ _('Accessibility statement') }}</h1>
 
   <h2 class="govuk-heading-m">{{ _('Using this service') }}</h2>
   <p class="govuk-body">
+    {% trans %}This accessibility statement applies to <a class="govuk-link" href="https://checklegalaid.service.gov.uk/">https://checklegalaid.service.gov.uk/</a>.{% endtrans %}
```

Comment:
I don't see any welsh translation for this. Will that be coming later?

Reply:
Yes, there are a few translations missing.  Josh will get them and we'll update them in a new ticket.